scripts/ensure-test-workdir.js

```diff
@@ -2,12 +2,16 @@ const fs = require('fs')
 
 const path = './test-mould'
 const RESOLVER_PATH = `${path}/resolvers.ts`
+const SETUP_PATH = `${path}/setup.ts`
 const SYMLINK_MOULD_PATH = './.mould'
 
 fs.existsSync(path) || fs.mkdirSync(path)
 if (!fs.existsSync(RESOLVER_PATH)) {
     fs.writeFileSync(RESOLVER_PATH, `export default {}`)
 }
+if (!fs.existsSync(SETUP_PATH)) {
+    fs.writeFileSync(SETUP_PATH, `export default () => ({})`)
```

Comment:
You provide the default function with no arguments, but use this function with an argument.
While building, `tsc` detects is as an error 

```ts
mould.ts:81:35 - error TS2554: Expected 0 arguments, but got 1.

81 const extentions: InsType = setup(MouldInstance) as any
                                     ~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.

```

Reply:
Sorry, didn't notice here